keyboards/orthodox/keymaps/rfvizarra/keymap.c

```diff
@@ -0,0 +1,206 @@
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,  _______, _______ \
+  ),
+
+  [_NAV2] =  LAYOUT( \
+    _______, _______, _______, _______, _______, _______,                                                                _______, _______, KC_PGUP, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______,          _______, _______,          _______, _______,          _______, KC_HOME, KC_PGDN, KC_END,  _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+  ),
+
+  [_MEDIA] =  LAYOUT( \
+    _______, _______, _______, _______, _______, _______,                                                                _______, _______, KC_VOLU, _______, _______, _______, \
+    _______, _______, _______, _______, _______, _______,          _______, _______,          _______, _______,          _______, KC_MNXT, KC_VOLD, KC_MPLY, _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+  ),
+
+  [_ADJUST] =  LAYOUT(                                                  \
+    _______, RESET,   _______, _______, _______, _______,                                                                _______, _______, _______, _______, _______, _______,  \
+    _______, _______, _______, AU_ON,   AU_OFF,  AG_NORM,          _______, _______,          _______, _______,          AG_SWAP, QWERTY , COLEMAK, DVORAK,  _______, _______, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY  \
+  )
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+  case QWERTY:
+    if (record->event.pressed) {
+#ifdef AUDIO_ENABLE
+      PLAY_SONG(tone_qwerty);
+#endif
+      persistent_default_layer_set(1UL<<_QWERTY);
+    }
+    return false;
+    break;
+  case COLEMAK:
+    if (record->event.pressed) {
+#ifdef AUDIO_ENABLE
+      PLAY_SONG(tone_colemak);
+#endif
+      persistent_default_layer_set(1UL<<_COLEMAK);
+    }
+    return false;
+    break;
+  case DVORAK:
+    if (record->event.pressed) {
+#ifdef AUDIO_ENABLE
+      PLAY_SONG(tone_dvorak);
+#endif
+      persistent_default_layer_set(1UL<<_DVORAK);
+    }
+    return false;
+    break;
+  case LOWER:
```

Comment:
This is optional, but you can support all of this with the new `update tri layer state` function, so that you can use any layer key.  The caveat, is that the 3rd layer is no longer accessible without using the tri layer. 

So you could replace the RAISE/LOWER with MO() functions, or even LT(x, kc) keys 
(like this: https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L41-L42)

And then add this block: 
```c
uint32_t layer_state_set_user(uint32_t state) {
  state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
  state = update_tri_layer_state(state, _LOWER, _NAV, _NAV2);
  return state;
}
```